value.go

```diff
@@ -139,7 +139,7 @@ func (v *MrbValue) GCProtect() {
 // when this value is a proc.
 func (v *MrbValue) SetProcTargetClass(c *Class) {
 	proc := C._go_mrb_proc_ptr(v.value)
-	proc.target_class = c.class
+	proc.e = *(*[8]byte)(unsafe.Pointer(&c.class))
```

Comment:
yeah this isn't going to work, unless 2.0 has reverted some of the structs this will fail; the target_class notion has completely been changed in 1.4.

I need to read the 2.0 source before I can comment further.